src/modelbench/run.py

```diff
@@ -144,15 +152,27 @@ def find_suts_for_sut_argument(sut_args: List[str]):
     return suts
 
 
-def get_benchmark(version: str, locale: Optional[Locale]) -> BenchmarkDefinition:
+def ensure_ensemble_annotators_loaded():
+    try:
+        from modelgauge.private_ensemble_annotator_set import EnsembleAnnotatorSet, ensemble_secrets
+
+        private_annotators = EnsembleAnnotatorSet(secrets=ensemble_secrets(load_secrets_from_config()))
+        modelgauge.tests.safe_v1.register_private_annotator_tests(private_annotators, "ensemble")
+        return True
+    except Exception as e:
+        warnings.warn(f"Can't load private ensemble annotators: {e}")
```

Comment:
I take it that this failure isn't a blocker for the run?

Reply:
Sometimes yes, sometimes no. I started out always throwing an exception, but sometimes you want to just know if they are available.